.github/workflows/e2e-minikube-helm.yml

```diff
@@ -27,3 +27,8 @@ jobs:
         export PLATFORM=minikube
         export INSTALLER=helm
         yarn test --coverage=false --forceExit --testRegex=test/e2e/e2e.test.ts
+    - uses: actions/upload-artifact@v2
+      if: ${{ always() }}
+      with:
+        name: minikube-updates-artifacts
```

Comment:
what about `Upload artifacts` ?

Reply:
It is a zip name, will be better to use minikube-helm-artifacts. WDYT?